pkgs/development/interpreters/metamath/default.nix

```diff
@@ -1,16 +1,18 @@
 { stdenv, fetchFromGitHub, autoreconfHook }:
 
-stdenv.mkDerivation {
+let
+  version = "0.194";
+in stdenv.mkDerivation {
```

Comment:
```suggestion
stdenv.mkDerivation rec {
```

Reply:
I'm not a huge fan of using `rec` for just a single entry, it feels very overkill to me

Reply:
It is how most packages are done.